lib/omniauth/strategies/openid_connect.rb

```diff
@@ -335,10 +337,10 @@ def verify_id_token!(id_token)
       class CallbackError < StandardError
         attr_accessor :error, :error_reason, :error_uri
 
-        def initialize(error, error_reason = nil, error_uri = nil)
-          self.error = error
-          self.error_reason = error_reason
-          self.error_uri = error_uri
+        def initialize(data)
```

Comment:
Why do you prefer to use `Hash` instead of keyword arguments?

```ruby
def initialize(error:, reason: nil, uri: nil)
  self.error = error
  self.error_reason = reason
  self.error_uri = uri
end
```

this way you can validate the names of passed values and make sure there's mandatory `error` argument

Reply:
the problem here ist, that we need a single param to construct the error.
With multiple params the `raise CallbackError, error: "...", reason: "..."` don't work.